src/App.tsx

```diff
@@ -1,19 +1,125 @@
+/* eslint-disable react/no-unused-state */
 import React from 'react';
-import './App.css';
-
+import './App.scss';
+import todos from './api/todos';
 import users from './api/users';
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
-};
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './types';
+
+const preparedTodos: Todo[] = todos.map((todo) => {
+  return {
+    ...todo,
+    user: users.find(user => user.id === todo.userId) || null,
+  };
+});
+
+interface State {
+  todos: Todo[];
+  todoTitle: string;
+  userId: number;
```

Comment:
And it'd be better to set it as the null at the start
```suggestion
  userId: number | null;
```